cli/cmd/create/create.go

```diff
@@ -97,6 +98,9 @@ func (c *Create) RunCallback(ctx *clicontext.CLIContext, cb func(service *riov1.
 	}
 
 	service.Namespace, service.Name = stack.NamespaceAndName(ctx, service.Name)
+	if strings.Contains(service.Name, ":") {
```

Comment:
nit: `service.Name, service.Spec.Version = kv.Split(service.Name, ":")` is enough